longevity_test.py

```diff
@@ -269,10 +269,18 @@ def test_user_batch_custom_time(self):
         :param batch_size: Number of stress commands to run together in a batch.
         """
 
+        customer_profiles = self.params.get('cs_user_profiles')
+        if not customer_profiles:
+            TestFrameworkEvent(source="test_user_batch_custom_time",
+                               message="The Longevity test_user_batch_custom_time cannot run without 'cs_user_profiles' parameter. aborting.",
+                               severity=Severity.ERROR).publish()
+            return
         self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(), tester_obj=self)
 
         batch_size = self.params.get('batch_size')
-
+        user_profile_table_count = self.params.get('user_profile_table_count')  # pylint: disable=invalid-name
+        # Creating extra table during batch only for a high number of tables (>500).
+        create_extra_tables = True if int(user_profile_table_count) > 500 else False
```

Comment:
This approach looks strange. 
Why exactly 500? What if later we will have 400 and 800 cases?

So, need either make it be configurable or describe the reasoning for it here.
Also, coding of this particular condition can be simplified to the following:

```
create_extra_tables = int(user_profile_table_count) > 500
```

Reply:
In the original 5000 table case, we created all 5000 tables upfront

we wanted few to be created in run time, via c-s as well 

In each cycle we added 4 of them, I don't know how it became just 1

Reply:
And yes if you want to take it out for some tests, it should be configurable.

No one's gonna remember what's this logic you did here in two weeks

Reply:
ok, fixed by:
![image](https://github.com/user-attachments/assets/2a421ffe-2427-4c96-8597-c83555d1ac5d)